base_business_document_import/models/business_document_import.py

```diff
@@ -720,7 +720,8 @@ def _match_tax(
                 ("unece_due_date_code", "=", tax_dict["unece_due_date_code"]),
                 ("unece_due_date_code", "=", False),
             ]
-        taxes = ato.search(domain, order="unece_due_date_code")
+        taxes = ato.search(domain)
```

Comment:
Why the order attribute is removed?

Reply:
@fshah-initos because the records will be sorted with _get_unece_due_date_type_code, see the next line